data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/firststrike/firststrike_ability.cfg

```diff
@@ -0,0 +1,101 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [firststrike]
+##
+# Actions:
+# Give bob a firststrike ability
```

Comment:
(for multiple tests) Why it does 2 [test_do_attack_by_id] between same units. Looks like the only thing second tests is [modify_unit] affecting all weapons.

Reply:
They're using weapon 0 and then weapon 1.

Reply:
But both of them have first strike.

Reply:
The filter gives `id = bob`.